test-suite/generated-src/jni/NativeCppException.cpp

```diff
@@ -29,7 +29,7 @@ CJNIEXPORT jint JNICALL Java_com_dropbox_djinni_test_CppException_00024CppProxy_
     } JNI_TRANSLATE_EXCEPTIONS_RETURN(jniEnv, 0 /* value doesn't matter */)
 }
 
-CJNIEXPORT jobject JNICALL Java_com_dropbox_djinni_test_CppException_get(JNIEnv* jniEnv, jobject /*this*/)
+CJNIEXPORT jobject JNICALL Java_com_dropbox_djinni_test_CppException_00024CppProxy_get(JNIEnv* jniEnv, jobject /*this*/)
```

Comment:
Could you please explain the 00024CppProxy?

Reply:
get() is a static function declared here in exception.djinni
```
cpp_exception = interface +c {
    throw_an_exception(): i32;
    static get(): cpp_exception;
}
```
This change moved the Java declarations of static functions that are implemented in C++ into the CppProxy inner class. "_00024" corresponds to the "$" symbol, which sits between inner and outer Java classes' names in the native name of a Java method that belongs to an inner class.